spec/toml-spec.coffee

```diff
@@ -18,53 +18,80 @@ describe "TOML grammar", ->
+    expect(tokens[1]).toEqual value: "I'm ", scopes: ["source.toml", "string.quoted.double.toml"]
+    expect(tokens[2]).toEqual value: "\\n", scopes: ["source.toml", "string.quoted.double.toml", "constant.character.escape.toml"]
+    expect(tokens[3]).toEqual value: " escaped", scopes: ["source.toml", "string.quoted.double.toml"]
+    expect(tokens[4]).toEqual value: '"', scopes: ["source.toml", "string.quoted.double.toml", "punctuation.definition.string.end.toml"]
+
+    {tokens} = grammar.tokenizeLine("'I am not \\n escaped'")
+    expect(tokens[0]).toEqual value: "'", scopes: ["source.toml", "string.quoted.single.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: 'I am not \\n escaped', scopes: ["source.toml", "string.quoted.single.toml"]
+    expect(tokens[2]).toEqual value: "'", scopes: ["source.toml", "string.quoted.single.toml", "punctuation.definition.string.end.toml"]
+
+  it "tokenizes multiline strings", ->
+    {tokens} = grammar.tokenizeLine('"""\nI am a\\\nstring"""')
+    expect(tokens[0]).toEqual value: '"""', scopes: ["source.toml", "string.quoted.double.block.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: '\nI am a', scopes: ["source.toml", "string.quoted.double.block.toml"]
+    expect(tokens[2]).toEqual value: '\\', scopes: ["source.toml", "string.quoted.double.block.toml", "constant.character.escape.toml"]
+    expect(tokens[3]).toEqual value: '\nstring', scopes: ["source.toml", "string.quoted.double.block.toml"]
+    expect(tokens[4]).toEqual value: '"""', scopes: ["source.toml", "string.quoted.double.block.toml", "punctuation.definition.string.end.toml"]
+
+    {tokens} = grammar.tokenizeLine("'''\nI am a\\\nstring'''")
+    expect(tokens[0]).toEqual value: "'''", scopes: ["source.toml", "string.quoted.single.block.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: '\nI am a\\\nstring', scopes: ["source.toml", "string.quoted.single.block.toml"]
+    expect(tokens[2]).toEqual value: "'''", scopes: ["source.toml", "string.quoted.single.block.toml", "punctuation.definition.string.end.toml"]
 
   it "tokenizes booleans", ->
     {tokens} = grammar.tokenizeLine("true")
     expect(tokens[0]).toEqual value: "true", scopes: ["source.toml", "constant.language.boolean.true.toml"]
+
     {tokens} = grammar.tokenizeLine("false")
     expect(tokens[0]).toEqual value: "false", scopes: ["source.toml", "constant.language.boolean.false.toml"]
 
-  it "tokenizes numbers", ->
-    {tokens} = grammar.tokenizeLine("123")
-    expect(tokens[0]).toEqual value: "123", scopes: ["source.toml", "constant.numeric.toml"]
-
-    {tokens} = grammar.tokenizeLine("-1")
-    expect(tokens[0]).toEqual value: "-1", scopes: ["source.toml", "constant.numeric.toml"]
-
-    {tokens} = grammar.tokenizeLine("3.14")
-    expect(tokens[0]).toEqual value: "3.14", scopes: ["source.toml", "constant.numeric.toml"]
+  it "tokenizes integers", ->
```

Comment:
Do we need separate `it` blocks for ints and floats, considering that they're both tokenized the same?